ambry-cloud/src/test/resources/azure-test.properties

```diff
@@ -9,9 +9,8 @@
 # under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
 # CONDITIONS OF ANY KIND, either express or implied.
 #
-azure.storage.connection.string=<storage-account-credentials>
-cosmos.endpoint=<cosmos-url>
-cosmos.collection.link=/dbs/ambry-metadata/colls/blob-metadata
-cosmos.key=<cosmos-key>
-cosmos.deleted.container.collection.link=/dbs/ambry-metadata-main/colls/deleted-container-test
-cosmos.direct.https=true
+azure.storage.connection.string=DefaultEndpointsProtocol=https;AccountName=wus2ambryblobstore1;AccountKey=vqQbP8s6I68IGlyYeqyv1dAwsh3AXXlD+QwgGTVhbU5NEMzE6Tucv9SxS2/6AzB/8VDHjaZmIYV9MWUCWaAqTg==;EndpointSuffix=core.windows.net
```

Comment:
Ouch!  Obviously remove this.  We should also change the EI keys since they've been exposed.